composer.json

```diff
@@ -50,17 +53,18 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.3",
         "composer/xdebug-handler": "^2.0",
-        "phan/phan": "^5.0",
         "friendsofphp/php-cs-fixer": "^3.0",
-        "vimeo/psalm": "^4.0",
+        "guzzlehttp/guzzle": "^7.3",
+        "guzzlehttp/psr7": "^2.0@RC",
+        "mockery/mockery": "^1.4",
```

Comment:
I installed mockery in order to test some of the interactions of `SpanBuilder`. IMO it's a much better mocking library than the one that comes with `PHPUnit`.